components/create_comment/create_comment.jsx

```diff
@@ -548,7 +559,29 @@ export default class CreateComment extends React.PureComponent {
         this.draftsForPost[this.props.rootId] = updatedDraft;
     }
 
+    handleMouseUp = (e) => {
+        e.persist();
+        const caretPosition = Utils.getCaretPosition(e.target);
+        const target = e;
```

Comment:
Shouldn't this be `const target = e.target;`? That might get rid of the need to persist the event as well

Reply:
@hmhealey it's actually me storing the event. I should have named it appropriately, something like `targetEvent`. But after implementing your last feedback on this PR, that's now deleted.